index.html

```diff
@@ -25,6 +27,45 @@
 			</li>
 		</ul>
 	</nav>
+
+	<section class="hero">
+		<section class="container">
+			<article id="hero-text">
+				<h1>Josh Selby</h1>
+				<h3>Full stack web developer</h3>
+				<h6>guitar player, photographer, car mechanic..</h6>
+			</article>
+
+			<a id="hero-arrow" href="#about-text"><div>▼</div></a>
```

Comment:
Pasting this arrow in natively isn't working right for me. Try and use an HTML entity if there is one, or maybe 'draw' the shape in CSS

Reply:
fixed